pkg/commands/process/settings/rules/javascript/express/insecure_allow_origin.yml

```diff
@@ -0,0 +1,62 @@
+patterns:
+  - pattern: |
+      res.$<METHOD>($<ORIGIN>, $<USER_INPUT>)
+    filters:
+      - variable: METHOD
+        values:
+          - set
+          - header
+      - variable: ORIGIN
+        values:
+          - '"Access-Control-Allow-Origin"'
+          - '"access-control-allow-origin"'
+          - "'Access-Control-Allow-Origin'"
+          - "'access-control-allow-origin'"
+      - variable: USER_INPUT
+        detection: javascript_express_insecure_allow_origin_user_input
+  - pattern: |
+      res.writeHead($<_>, $<INSECURE_CORS_HASH>)
+    filters:
+      - variable: INSECURE_CORS_HASH
+        detection: javascript_express_insecure_allow_origin_insecure_cors_hash
+languages:
+  - javascript
+auxiliary:
+  - id: javascript_express_insecure_allow_origin_user_input
+    patterns:
+      - req.params
+      - req.query
+      - req.body
+      - req.cookies
+      - req.headers
+  - id: javascript_express_insecure_allow_origin_insecure_cors_hash
+    patterns:
+      - pattern: |
+          { $<ORIGIN>: $<USER_INPUT> }
+        filters:
+          - variable: ORIGIN
+            values:
+              - '"Access-Control-Allow-Origin"'
+              - '"access-control-allow-origin"'
+              - "'Access-Control-Allow-Origin'"
+              - "'access-control-allow-origin'"
```

Comment:
We could use a regex here, might simplify that a bit
see https://github.com/Bearer/bearer/pull/575